docs/testing.rst

```diff
@@ -9,15 +9,15 @@ Front-end testing
 =================
 
 Bedrock runs a suite of front-end `Jasmine`_ behavioral/unit tests, which use
-`Karma`_ as a test runner. We also have a suite of functional tests using
+`Jasmine Browser Runner`_ as a test runner. We also have a suite of functional tests using
 `Selenium`_ and `pytest`_. This allows us to emulate users interacting with a
 real browser. All these test suites live in the ``tests`` directory.
 
 The ``tests`` directory comprises of:
 
 * ``/functional`` contains pytest tests.
 * ``/pages`` contains Python page objects.
-* ``/unit`` contains the Jasmine tests and Karma config file.
+* ``/unit`` contains the Jasmine tests and Jasmine Browser Runner config file.
 
 Installation
```

Comment:
Is it worth adding a note in the docs that people will need both `geckodriver` and `chromedriver` installed locally before running the tests?

There's already a bit under functional testing that talks about how to install geckodriver: https://bedrock.readthedocs.io/en/latest/testing.html#running-functional-tests

Perhaps we could re-purpose this at the top of the doc, saying both `geckodriver` and `chromedriver` are a dependency for running both JS and functional tests locally?

Reply:
Yeah, good call out.